WordPress/src/main/res/values-de/strings.xml

```diff
@@ -554,11 +554,11 @@ Language: de
     <string name="stats_no_data_for_period">Keine Daten für diesen Zeitraum</string>
     <string name="preference_strip_image_location">Positionsangabe von Medien entfernen</string>
     <string name="stats_cannot_be_started">Wir können die Statistiken momentan nicht öffnen. Bitte versuche es später erneut</string>
-    <string name="giphy_powered_by_giphy">Bereitgestellt von GIPHY</string>
```

Comment:
Maybe I didn't mention this explicitly. We shouldn't change any of the translation files. It should all go through the translation service. You should just change the names of the string items in the basic strings.xml (and you can also delete the translations of the original strings). I think (I'm not sure here) they'll get overriden by the service result anyway.

Reply:
Just to be sure before I remove those changes, should I also revert the translated strings which only got renamed? Or this revert applies only to the strings in which the content was changed?

Reply:
Thanks for the feedback! I just deleted all translation strings as suggested and left the renaming and the Tenor replacement only at the original `strings.xml` file, would be fine that way? Here's the commit: 8eabca9ae83f3936c3ae438aca70e22e3908dca5 . Thanks!